docs/source/markdown/podman-network-create.1.md

```diff
@@ -118,6 +126,9 @@ Additionally the `bridge` driver supports the following options:
 - `com.docker.network.bridge.name`: This option assigns the given name to the created Linux Bridge
 - `com.docker.network.driver.mtu`: Sets the Maximum Transmission Unit (MTU) and takes an integer value.
 - `vrf`: This option assigns a VRF to the bridge interface. It accepts the name of the VRF and defaults to none. Can only be used with the Netavark network backend.
+- `mode`: This option sets the specified bridge mode on the interface. Defaults to `managed`. Supported values:
+  - `managed`: Podman creates and deleted the bridge. It will also setup sysctls and firewall rules for it.
```

Comment:
Maybe add that the firewall rules are added to MASQUERADE/NAT the traffic and for port forwarding DNAT rules are added. And then in the unmanaged case make clear no firewall rules are added which means no DNAT rules for port forwarding as well which might surprise users if they add ports but they don't work.

Reply:
good idea. added.